src/Mvc/Mvc.ViewFeatures/src/Infrastructure/ArrayBufferWriter.cs

```diff
@@ -0,0 +1,181 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
License is different because it came from corefx?

Reply:
Yeah. I think the plan is for the type to go away. I'll update the license to keep it in sync with the rest of the codes.